README.md

```diff
@@ -107,12 +108,17 @@ ubi extension.
       registry if you are not running a local registry.
    1. run `docker push localhost:5000/test-builder` to push the builder to the
       local registry or push to a public registry if desired.
+
+1. Pull the run images needed by the extension. These are not pulled automatically
+   when you build the application.
+   1. run `docker pull quay.io/midawson/ubi8-paketo-run-nodejs-18`
+   1. run `docker pull quay.io/midawson/ubi8-paketo-run-nodejs-16`
```

Comment:
@pacostas I think this might be being added back by accident.  This step should no longer be required with the latest version of pack.

Reply:
Didnt know, thank you! Removing it.